src/main/java/com/metamx/common/guava/LimitedYieldingAccumulator.java

```diff
@@ -18,6 +18,7 @@
 
 /**
  */
+@Deprecated
```

Comment:
This class uses volatile internally, for counter, but it's not thread-safe, because volatile is used improperly. This class is not used neither in druid nor in metamx codebase. (There is a private inner class with the same name in LimitedSequence, which is used; apparently this top-level LimitedYieldingAccumulator is just forgotten to be removed during some refactoring.)

Since java-util has version 0.x I think it's ok to remove this class and update minor version (27 -> 28). What do you think?